internal/serve/httphandler/user_handler.go

```diff
@@ -284,7 +291,7 @@ func (h UserHandler) GetAllUsers(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	users, err := h.AuthManager.GetAllUsers(ctx, token)
+	users, err := h.AuthManager.GetAllUsers(ctx, token, queryParams)
```

Comment:
It's easier to use a sort on the users here instead of doing it at the `stellar-auth` package level (query), IMO. I suggest you move [this block of code](https://github.com/stellar/stellar-disbursement-platform-backend/pull/104/files#diff-06b2e13ef7272f0d49e79f045d25b1d3238592744759dce66c48fffab7449928R21-R25) to here.